k8s/src/test/scala/io/buoyant/k8s/EndpointsNamerTest.scala

```diff
@@ -86,14 +111,13 @@ class EndpointsNamerTest extends FunSuite with Awaits {
         fail(s"unexpected request: $req")
     }
     val api = v1.Api(service)
-    val namer = new EndpointsNamer(Path.read("/test"), api.withNamespace(_))
+    val namer = new EndpointsNamer(Path.read("/test"), api.withNamespace)
 
     @volatile var state: Activity.State[NameTree[Name]] = Activity.Pending
-    val obs = namer.lookup(Path.read("/srv/thrift/sessions")).states respond { s =>
+    val _ = namer.lookup(Path.read("/srv/thrift/sessions")).states respond { s =>
```

Comment:
Ah, ok.  We should discuss introducing new stylistic checks.  Throughout our codebase (and certainly finagle), discarded values are used idiomatically.  I'm not opposed to it, but i"m not sure how much value this actually adds if we are diligent about annotating return types